openquake/hazardlib/tests/geo/surface/kite_fault_test.py

```diff
@@ -899,14 +897,15 @@ def test_mesh_creation_sa(self):
         # FIXME
         # self.assertEqual(np.sum(np.sum(idx)), 207)
 
+        breakpoint()
```

Comment:
Looks like you committed the breakpoint accidentally

Reply:
Fixed. Grazie!